src/lib/apiTypesV2/BatchQuery.cpp

```diff
@@ -53,38 +53,6 @@ BatchQuery::~BatchQuery()
 
 /* ****************************************************************************
 *
-* BatchQuery::check - 
-*/
-std::string BatchQuery::check(ConnectionInfo* ciP, RequestType requestType)
```

Comment:
This needs a careful examination... I have the feeling that righ now most of the checking is done "inline" as the parsing is done. In that case, the strategy of "parsing all, them check" would not easy to apply as it would require big changes in the code. And actually I'm not sure if it is the best one... I mean, maybe is better to early detect error conditions with inline checking.

Note the *maybe* :) I mean, I'm not sure which option is better. What I'm sure is that we would need time to discuss so I'm leave that appart from that PR.

Thus, in sum, what I can do is to keep the code of these methods under `#if 0` instead of removing them, with a FIXME comment about "a discussion is pending if we should implement a check-as-we-parse or parse-all-check-after strategy in NGSIv2 parsing"

(Note the "in NGSIv2 parsing". NGSIv1 parsing doen't deserves changes in the strategy or big changes, from my point of view)